common/primitives/constants.go

```diff
@@ -50,3 +50,8 @@ const (
 const (
 	DefaultHistoryMaxAutoResetPoints = 20
 )
+
+const (
+	// WorkflowIDPrefix ID prefix for schedules that are implemented by a workflow
+	WorkflowIDPrefix = "temporal-sys-scheduler:"
```

Comment:
the name `WorkflowIDPrefix` only makes sense in the `scheduler` package for context. here it should be `ScheduleWorkflowIDPrefix`